app/assets/stylesheets/classified_listings.scss

```diff
@@ -556,6 +556,11 @@
     theme-color,
     $white
   );
+  @include themeable(
+    background-color,
+    theme-container-accent-background,
+    $white
```

Comment:
This doesn't seem right 🤔Both default value for color and background-color are `$white`

Reply:
@Glennmen thanks for spotting this! I've fixed this now 👍